examples/40_advanced/example_resampling.py

```diff
@@ -89,26 +90,39 @@
 # =============================
 # It is also possible to use
 # `scikit-learn's splitter classes <https://scikit-learn.org/stable/modules/classes.html#splitter
-# -classes>`_ to further customize the outputs.
+# -classes>`_ to further customize the outputs. In case one needs to have 100% control over the
+# splitting, it is possible to use
+# `scikit-learn's PredefinedSplit <https://scikit-learn.org/stable/modules/generated/
+# sklearn.model_selection.PredefinedSplit.html>`_.
 
+############################################################################
+# Exemplary use of a predefined split. We split the training data by the first
```

Comment:
Do you mean: An example of a predefined split?

Reply:
Yes, updated the example.